File: php/program/Civitas.php
<?php

include_once("Mahasiswa.php");

class Civitas extends Mahasiswa{

    private $university;
    private $email;
    public function __construct($id, $name, $gender ,$nim, $university, $prodi, $fakultas, $nik, $email){
        $this->id = $id;
        $this->name = $name;
        $this->nim = $nim;
        $this->university = $university;
        $this->prodi = $prodi;
        $this->fakultas = $fakultas;
        $this->imgPath = "default.jpg";
        $this->gender = $gender;
        $this->nik = $nik;
        $this->email = $email;
    }

    public function setUniversity($university){
        $this->university = $university;
    }

    public function setEmail($email){
        $this->email = $email;
    }

    public function getUniversity(){
        return $this->university;
    }

    public function getEmail(){
        return $this->email;
    }
}

?>

File: python/program/Civitas.py
from Mahasiswa import *

class Civitas(Mahasiswa):
    
    def __init__(self, name, nim, gender, university, prodi, fakultas, nik, email):
        self._name = name
        self._nim = nim
        self._gender = gender
        self._prodi = prodi
        self._fakultas = fakultas
        self._nik = nik
        self._university = university
        self._email = email
    
    def setUniversity(self, university):
        self._university = university
    
    def setEmail(self, email):
        self._email = email
    
    def getUniversity(self):
        return self._university
    
    def getEmail(self):
        return self._email